lib/ansible/modules/uri.py

```diff
@@ -364,8 +364,13 @@
 # The return information includes all the HTTP headers in lower-case.
 content:
   description: The response body content.
-  returned: status not in status_code or return_content is true
+  returned: status not in status_code or return_content is true and reported Content-type is not "application/json"
   type: str
+  sample: ""
+json:
+  description: The response body content.
+  returned: if the reported Content-type is "application/json"
+  type: complex
```

Comment:
complex should not be used anymore

Reply:
As @jborean93 recommended, I changed the type to raw.

Reply:
type would be 'string' in case of really returning JSON